SU-SWS/stanford_sites_helper
# Update hook_requirements() link to correct SNOW form

# READY FOR REVIEW

# Summary
- We missed one

# Needed By (Date)
- Whenever; I'm not deploying this change tomorrow, and we're moving off of AFS.

# Criticality
- How critical is this PR on a 1-10 scale? 0.001/10


# Steps to Test

1. Go to the previous link
2. See nothing
3. Go to the new link
4. See the "Increase storage limit on Stanford Sites Drupal Self-Service Websites" form

# Affected Projects or Products
- Sites 1.x

# Associated Issues and/or People
- JIRA ticket: https://asjira.stanford.edu/jira/browse/DRPL-743
- Other PRs: #35 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
